deepchem/__init__.py

```diff
@@ -3,7 +3,7 @@
 """
 
 # If you push the tag, please remove `.dev`
-__version__ = '2.4.0'
+__version__ = '2.5.0.dev'
```

Comment:
We have already pushed the 2.4.0 tag, so I set the dev version of the next release.